nano/node/active_transactions.cpp

```diff
@@ -579,6 +588,80 @@ void nano::active_transactions::erase (nano::block const & block_a)
 	}
 }
 
+bool nano::active_transactions::should_flush ()
+{
+	bool result (false);
+	counter.trend_sample ();
+	size_t minimum_size (1);
+	std::unique_lock<std::mutex> counter_lock (counter.mutex);
+	if (counter.rate == 0)
+	{
+		minimum_size = 4;
```

Comment:
Isn't default minimum too low? Why drop from 4 active elections on live/beta?

Reply:
minimum moved to the counter multiplier of 512, but added check for test_network to drop it down to 4 to make testing easier

Reply:
Seems reasonable